integrations/fullstory/test/index.test.js

```diff
@@ -179,5 +186,44 @@ describe('FullStory', function() {
         );
       });
     });
+
+    describe('#page', function() {
+      beforeEach(function() {
+        analytics.stub(window.FS, 'event');
+      });
+
+      it('should not track unnamed pages by default', function() {
+        analytics.page();
+        analytics.didNotCall(window.FS.event);
+      });
+
+      it('should track unnamed pages if enabled', function() {
+        fullstory.options.trackAllPages = true;
+        analytics.page();
+        analytics.called(window.FS.event, 'Loaded a Page');
+      });
+
+      it('should not track named pages by default', function() {
+        analytics.page('Name');
+        analytics.didNotCall(window.FS.event);
+      });
+
+      it('should track named pages if enabled', function() {
+        fullstory.options.trackNamedPages = true;
+        analytics.page('Name');
+        analytics.called(window.FS.event, 'Viewed Name Page');
+      });
+
+      it('should not track categorized pages by default', function() {
+        analytics.page('Category', 'Name');
+        analytics.didNotCall(window.FS.event);
+      });
+
+      it('should track categorized pages if enabled', function() {
+        fullstory.options.trackCategorizedPages = true;
+        analytics.page('Category', 'Name');
+        analytics.called(window.FS.event, 'Viewed Category Page');
```

Comment:
@CarlosMecha Note this differs from the expected result in the original Webonise PR, e.g that Webonise test would have failed as written: https://github.com/webonise/analytics.js-integrations/blob/DWEB-3/integrations/fullstory/test/index.test.js#L219

Do you remember if the customer wanted "Viewed Category Page" or "Viewed Category Name Page"?

Reply:
They only mentioned `Viewed <Pagename> Page`... How about we do `Viewed Category <Name> Page`?

Reply:
Actually, my bad - `Viewed <Category> <Name> Page` would be handled by `trackNamedPages`, so long as the payload includes both a name and a category. So I think we should leave as is.